front/components/assistant/AssistantBrowser.tsx

```diff
@@ -199,7 +215,23 @@ export function AssistantBrowser({
               description={agent.description}
               variant="minimal"
               onClick={() => handleAssistantClick(agent)}
-              onActionClick={() => setAssistantIdToShow(agent.sId)}
+              actionElement={
+                <>
+                  {/* Theses 2 divs are an ugly hack to align the button while making the dropdown menu visible above the container, it has the size of the button hardcoded -- Let's fix it asap */}
```

Comment:
Could you please link it to a ticket? Or add a TODO?

Reply:
Yep !